src/main/scala/cromwell/webservice/ApiDataModels.scala

```diff
@@ -7,54 +7,67 @@ import cromwell.engine.backend.StdoutStderr
 
 import scala.annotation.meta.field
 
+@ApiModel(value = "WorkflowValidate")
+case class WorkflowValidateResponse
+(
+  @(ApiModelProperty@field)(required = true, value = "The validation of the workflow")
```

Comment:
Is this swagger stuff? If 'yes' I hate it even more than I thought. If 'no' why is this here?